ducktape/cluster/json.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, cluster_json=None, *args, **kwargs):
             {
               "externally_routable_ip": "192.168.50.151",
 
-              "ssh_config": {
+              "remote_command_config": {
```

Comment:
I'm fine with that, especially now that windows can run SSH commands (because it has an SSH server). I'll go ahead and move everything back to ssh_config.